tests/integration-tests/conftest.py

```diff
@@ -1081,11 +1086,64 @@ def vpc_stacks(cfn_stacks_factory, request):
             ],
         )
         template = NetworkTemplateBuilder(
-            vpc_configuration=vpc_config, default_availability_zone=availability_zones[0]
+            vpc_configuration=vpc_config,
+            default_availability_zone=availability_zones[0],
```

Comment:
Are we picking AZ[0] as the default AZ for all regions ?
Probably having a dictionary { "regX": "az_y" } would give us more flexibility

Reply:
making the stack multi-az is out of scope